operators/pkg/forge/cloudinit.go

```diff
@@ -49,6 +58,12 @@ type writefile struct {
 // CloudInitUserData forges the yaml manifest representing the cloud-init userdata configuration.
 func CloudInitUserData(nextcloudBaseURL, webdavUsername, webdavPassword string, publicKeys []string) ([]byte, error) {
 	config := userdata{
+		Users: []user{{
```

Comment:
User creation has somewhat broken ssh authorized keys configuration (both in case the user does already exists and in case it does not yet). Should be checked why these are always added to the root user.